qemu/tests/cfg/hpt_max_page_size.cfg

```diff
@@ -0,0 +1,28 @@
+- hpt_max_page_size: install setup image_copy unattended_install
+    type = hpt_max_page_size
+    only ppc64 ppc64le
+    only Linux
```

Comment:
ppc only support linux, I think we can remove this line

Reply:
Yes, you're right.